BigBaseV2/src/hooking.cpp

```diff
@@ -105,13 +110,21 @@ namespace big
 
 		m_send_net_info_to_lobby.enable();
 
+		m_chat_receive_hook.enable();
```

Comment:
recieve is incorrect, it should be receive

Reply:
It already is